x-pack/plugins/lens/public/visualizations/xy/xy_config_panel/axis_settings_popover.test.tsx

```diff
@@ -67,13 +67,23 @@ describe('Axes Settings', () => {
   });
 
   it('has the tickLabels switch on by default', () => {
-    const component = shallow(<AxisSettingsPopover {...props} />);
+    const component = shallow(
+      <AxisTicksSettings
+        axis={props.axis}
+        isAxisLabelVisible={props.areTickLabelsVisible}
+        updateTicksVisibilityState={jest.fn()}
```

Comment:
Can we add this to the `props` object?
So the previous code can be restored.

Reply:
Discussed with Marco offline, this is another component so I cant do what is proposed